### PR TITLE
PSMDB-901 Release notes for 5.0.2-1

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   source/conf.py
	new file:   source/release_notes/5.0.1-1.rst
	modified:   source/release_notes/index.rst

### CODE REVIEW

**Review by sandraromanchenko — APPROVED** [review]

**Review by igorsol — APPROVED** [review]
Please edit version in the header of this PR

File: source/release_notes/5.0.2-1.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- `Window operators <https://docs.mongodb.com/v5.0/release-notes/5.0/#window-operators>`_ allow operations on a specified span of documents known as window. `$setWindowFields <https://docs.mongodb.com/v5.0/reference/operator/aggregation/setWindowFields/#mongodb-pipeline-pipe.-setWindowFields>`_ is a new pipeline stage to operate with these documents.
+- `Auditing filters configuration at runtime <https://docs.mongodb.com/v5.0/release-notes/5.0/#auditing>`_
+- `Versioned API <https://docs.mongodb.com/v5.0/reference/versioned-api/>`_ allows specifying which API version your application communicating with MongoDB runs against. Versioned API detaches the application's lifecycle from that of the database. As a result you modify the application only to introduce new features instead of having to maintain compatibility with the new version of MongoDB.
+
+MongoDB 5.0.2 Community Edition also adds `new aggregation operators <onfluence.percona.com>`_ such as ``$count``, ``$dateAdd``, ``$dateDiff``, ``$dateSubtract``, ``$sampleRate`` и ``$rand`` and introduces `mongosh <https://docs.mongodb.com/v5.0/release-notes/5.0/#new-mongodb-shell--mongosh>`_ as a new shell client.
```
Comment (LenzGr) [inline source/release_notes/5.0.2-1.rst]: The URL to "aggregation operators" seems to be incomplete?
Also, there's a stray `и` in the enumeration.

File: source/release_notes/5.0.2-1.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+*Percona Server for MongoDB* 5.0.2-1
+================================================================================
+
+:Date: August 16, 2021
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/5.0/install/index.html>`_
+
+We are pleased to announce the release candidate of Percona Server for MongoDB 5.0.2-1. 
+
+Percona Server for MongoDB 5.0.2-1 is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.2 Community Edition. It  includes `all features of MongoDB 5.0.2 Community Edition <https://docs.mongodb.com/v5.0/release-notes/5.0/#5.0.2---aug-06--2021>`_ and provides :ref:`enterprise-level enhancements <compare>` for free.
```
Comment (LenzGr) [inline source/release_notes/5.0.2-1.rst]: Superfluous white space:
```suggestion
fully-compatible, drop-in replacement for MongoDB 5.0.2 Community Edition. It includes `all features of MongoDB 5.0.2 Community Edition <https://docs.mongodb.com/v5.0/release-notes/5.0/#5.0.2---aug-06--2021>`_ and provides :ref:`enterprise-level enhancements <compare>` for free.
```

File: source/release_notes/5.0.2-1.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.. _PSMDB-5.0.2-1:
+
+================================================================================
+*Percona Server for MongoDB* 5.0.2-1
```
Comment (LenzGr) [inline source/release_notes/5.0.2-1.rst]: Let's clearly mark it as "Release Candidate" here as well:
```suggestion
*Percona Server for MongoDB* 5.0.2-1 (Release Candidate)
```

File: source/release_notes/5.0.2-1.rst (hunk 1 above)
Comment (nastena1606) [reply source/release_notes/5.0.2-1.rst]: Yes, you are right, sorry about that. Fixed

File: source/release_notes/5.0.2-1.rst (hunk 2 above)
Comment (nastena1606) [reply source/release_notes/5.0.2-1.rst]: Fixed

File: source/release_notes/5.0.2-1.rst (hunk 3 above)
Comment (nastena1606) [reply source/release_notes/5.0.2-1.rst]: Added

File: source/release_notes/5.0.2-1.rst [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+:Date: August 16, 2021
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/5.0/install/index.html>`_
+
+We are pleased to announce the release candidate of Percona Server for MongoDB 5.0.2-1. 
+
+Percona Server for MongoDB 5.0.2-1 is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.2 Community Edition. It includes `all features of MongoDB 5.0.2 Community Edition <https://docs.mongodb.com/v5.0/release-notes/5.0/#5.0.2---aug-06--2021>`_ and provides :ref:`enterprise-level enhancements <compare>` for free.
+
+The following are most notable features introduced in MongoDB 5.0.2 Community Edition:
```
Comment (LenzGr) [inline source/release_notes/5.0.2-1.rst]:
```suggestion
Percona Server for MongoDB 5.0.2-1 includes all of the features of MongoDB 5.0 Community Edition. 

The most notable among these are:
```

File: source/release_notes/5.0.2-1.rst [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+- `Window operators <https://docs.mongodb.com/v5.0/release-notes/5.0/#window-operators>`_ allow operations on a specified span of documents known as window. `$setWindowFields <https://docs.mongodb.com/v5.0/reference/operator/aggregation/setWindowFields/#mongodb-pipeline-pipe.-setWindowFields>`_ is a new pipeline stage to operate with these documents.
+- `Versioned API <https://docs.mongodb.com/v5.0/reference/versioned-api/>`_ allows specifying which API version your application communicating with MongoDB runs against. Versioned API detaches the application's lifecycle from that of the database. As a result you modify the application only to introduce new features instead of having to maintain compatibility with the new version of MongoDB.
+
+MongoDB 5.0.2 Community Edition also adds `new aggregation operators <https://docs.mongodb.com/v5.0/release-notes/5.0/#new-aggregation-operators>`_ such as ``$count``, ``$dateAdd``, ``$dateDiff``, ``$dateSubtract``, ``$sampleRate`` and ``$rand``.
```
Comment (LenzGr) [inline source/release_notes/5.0.2-1.rst:23]:
```suggestion
Percona Server for MongoDB 5.0 also adds `new aggregation operators <https://docs.mongodb.com/v5.0/release-notes/5.0/#new-aggregation-operators>`_ such as ``$count``, ``$dateAdd``, ``$dateDiff``, ``$dateSubtract``, ``$sampleRate`` and ``$rand``.
```

**Review by LenzGr — CHANGES_REQUESTED** [review]

File: source/release_notes/5.0.2-1.rst (hunk 4 above)
Comment (sandraromanchenko) [reply source/release_notes/5.0.2-1.rst]: But these are features in MongoDB CE not in Percona Server for MongoDB, current wording seems to be more accurate and it was previously used:
https://www.percona.com/doc/percona-server-for-mongodb/4.0/release_notes/4.0.4-1.html
https://www.percona.com/doc/percona-server-for-mongodb/4.2/release_notes/4.2.0-1.html

File: source/release_notes/5.0.2-1.rst [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+================================================================================
+*Percona Server for MongoDB* 5.0.2-1 (Release Candidate)
+================================================================================
+
+:Date: August 16, 2021
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/5.0/install/index.html>`_
+
+We are pleased to announce the release candidate of Percona Server for MongoDB 5.0.2-1. 
+
+Percona Server for MongoDB 5.0.2-1 is an enhanced, open source, and highly-scalable database that is a
```
Comment (LenzGr) [inline source/release_notes/5.0.2-1.rst]:
```suggestion
Percona Server for MongoDB 5.0.2-1 is an enhanced, source-available, and highly-scalable database that is a
```

File: source/release_notes/5.0.2-1.rst (hunk 4 above)
Comment (LenzGr) [reply source/release_notes/5.0.2-1.rst]: Well, by these being features of MongoDB CE, they are features of PSMDB as well. I've updated the proposed change, hopefully that makes it more clear.

**Review by LenzGr — APPROVED** [review]
LGTM! Thanks a lot for incorporating the proposed changes.

**Review by sandraromanchenko — APPROVED** [review]

**Pull request merged**